website/identifiers/client.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+
+import furl
+import requests
+
+from . import utils
+
+
+class ClientError(Exception):
+    def __init__(self, message, response=None, code=None):
+        super(ClientError, self).__init__(message)
+        self.response = response
+        self.code = code
+
+
+class EzidClient(object):
+
+    BASE_URL = 'https://ezid.cdlib.org'
+
+    def __init__(self, username, password):
+        self.username = username
+        self.password = password
+
+    def _build_url(self, *segments, **query):
```

Comment:
Would it make sense to subclass this `BaseClient`? https://github.com/CenterForOpenScience/osf.io/blob/0b492db44504ea06c0d37eac768f0da92a5e451c/website/addons/googledrive/client.py#L16